sonic_data_client/mixed_db_client.go

```diff
@@ -492,7 +506,16 @@ func (c *MixedDbClient) tableData2Msi(tblPath *tablePath, useKey bool, op *strin
 		if tblPath.jsonTableKey != "" { // If jsonTableKey was prepared, use it
 			err = c.makeJSON_redis(msi, &tblPath.jsonTableKey, op, fv)
```

Comment:
>makeJSON_redis [](http://example.com/codeflow?start=11&length=14)

Do you need to check protobuf encoding before makeJson?

Reply:
Updated